gsdk/tests/rpc.rs

```diff
@@ -264,3 +265,50 @@ async fn test_runtime_wasm_blob_version_history() -> Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn test_original_code_storage() -> Result<()> {
+    let node = dev_node();
+
+    let salt = vec![];
+    let pid = ProgramId::generate(CodeId::generate(demo_messager::WASM_BINARY), &salt);
+
+    let signer = Api::new(Some(&node_uri(&node)))
+        .await?
+        .signer("//Alice", None)?;
+
+    signer
+        .upload_program(
+            demo_messager::WASM_BINARY.to_vec(),
+            salt,
+            vec![],
+            100_000_000_000,
+            0,
+        )
+        .await?;
+
+    let program = signer.api().gprog(pid).await?;
+    let last_block = signer
+        .api()
+        .rpc()
+        .block(None)
+        .await?
+        .unwrap()
```

Comment:
```suggestion
        .await??
```

hmm if this is possible?

Reply:
```
let rpc = signer.api().rpc();
```

Reply:
Done